graph/ui-icons/src/lib/framework-icons.stories.tsx

```diff
@@ -0,0 +1,24 @@
+import type { Meta, StoryObj } from '@storybook/react';
+import { Framework, frameworkIcons } from './framework-icons';
+
+const meta: Meta<typeof frameworkIcons> = {
+  component: () => (
+    <>
+      {Object.keys(frameworkIcons).map((key) => (
+        <>
+          <div>{key}</div>
+          <div className="h-10 w-10">
+            {frameworkIcons[key as Framework].image}
```

Comment:
Please handle when the `frameworkIcons` does not have the technology.

Reply:
done